lib/ctbl.gi

```diff
@@ -6266,11 +6203,35 @@ InstallGlobalFunction( CharacterTableOfNormalSubgroup,
               "#I   (now ", Length( classes ), ", after nec. splitting ",
               Length( classes ) + (p-1) * Length( err ), ")" );
       fi;
+      return fail;
+    fi;
 
-      Error( "tables in progress not yet supported" );
-#T !!
+    result:= Concatenation( "Rest(", Identifier( tbl ), ",",
+                            String( classes ), ")" );
+    ConvertToStringRep( result );
```

Comment:
I don't think calling `ConvertToStringRep` here is needed (but I see that this was just copied from the existing code, so it's OK to leave it in)